sshconfig

```diff
@@ -0,0 +1,80 @@
+Host bengine
+    HostName engine.bambi.ovh
+    User root
+    LocalForward 8080 127.0.0.1:80
+    StrictHostKeyChecking no
+    CheckHostIP no
+
+Host belk
+    HostName elk.bambi.ovh
+    User root
+    LocalForward 5601 127.0.0.1:5601
+    StrictHostKeyChecking no
+    CheckHostIP no
+
+Host bmoloch
+    HostName moloch.bambi.ovh
+    User root
+    LocalForward 8080 127.0.0.1:8080

```

Comment:
Isn't moloch running on 8005 by default?

Reply:
In our case it is actually neither nor :D

https://github.com/enowars/bambictf/blob/ed395504685c5d3f0d42639702cfd5ff18093858/ansible/roles/enomoloch/files/docker-compose.yml#L6